hedera-mirror-rest/topicmessage.js

```diff
@@ -202,6 +332,49 @@ const extractSqlFromTopicMessagesRequest = (topicId, filters) => {
   return utils.buildPgSqlObject(pgSqlQuery, pgSqlParams, order, limit);
 };
 
+const getTopicMessageTimestamps = async (topicId, filters, hasSequenceNumberForV2, limit, order) => {
+  const pgObject = hasSequenceNumberForV2
+    ? extractSqlForTopicMessagesLookup(topicId, filters, limit, order)
+    : await getSequenceNumberFromTopicMessageLookup(topicId, order, limit);
+
+  if (pgObject === null) {
+    return null;
+  }
+
+  const params = pgObject.params;
+  const query = pgObject.query;
+
+  const {rows} = await pool.queryQuietly(query, params);
+  if (rows.length === 0) {
+    throw new NotFoundError();
+  }
+
+  return _.isNil(rows) ? null : rows;
```

Comment:
rows won't be null or undefined
```suggestion
  return rows;
```